spoc/contacts.py

```diff
@@ -139,41 +247,60 @@ def _flip_unlabelled_contacts(self, df: DataFrame, start_index:Optional[int]=Non
         subsets = []
         for perm in permutations(range(start_index, end_index)):
             query = "<=".join([f"start_{i}" for i in perm])
-            subsets.append(df.query(query).rename(columns=self._generate_rename_columns(perm, start_index)))
+            subsets.append(
+                df.query(query).rename(
+                    columns=self._generate_rename_columns(perm, start_index)
+                )
+            )
         # determine which method to use for concatenation
         if isinstance(df, pd.DataFrame):
             result = pd.concat(subsets).sort_index()
             # this is needed if there are reads with equal start positions
-            result = result.loc[~result.index.duplicated(keep='first')]
+            result = result.loc[~result.index.duplicated(keep="first")]
         else:
-            result = dd.concat(subsets).reset_index()\
-                                        .sort_values("index")\
-                                        .drop_duplicates(subset=['index'])\
-                                        .set_index("index")
+            result = (
+                dd.concat(subsets)
+                .reset_index()
+                .sort_values("index")
+                .drop_duplicates(subset=["index"])
+                .set_index("index")
+            )
         return result
 
-    def _flip_labelled_contacts(self, df: DataFrame, label_values: List[str]) -> DataFrame:
+    def _flip_labelled_contacts(
+        self, df: DataFrame, label_values: List[str]
```

Comment:
No, everything is blacked now. If it fits in a line, black will do as above. Otherwise, it will split them.
Here is more info: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html